client/src/main/java/io/confluent/kafka/schemaregistry/AbstractSchemaProvider.java

```diff
@@ -39,6 +40,17 @@ public SchemaVersionFetcher schemaVersionFetcher() {
     return schemaVersionFetcher;
   }
 
+  @Override
+  public Optional<ParsedSchema> parseSchema(String schemaString,
```

Comment:
Let's make this a `default` method on the `SchemaProvider` interface

Reply:
Thanks for reviewing @rayokota 
Addressed.